cJSON.c

```diff
@@ -92,19 +92,19 @@ CJSON_PUBLIC(const char *) cJSON_GetErrorPtr(void)
     return (const char*) (global_error.json + global_error.position);
 }
 
-CJSON_PUBLIC(char *) cJSON_GetStringValue(const cJSON * const item) 
+CJSON_PUBLIC(char *) cJSON_GetStringValue(const cJSON * const item)
```

Comment:
Please don't mix large-scale whitespace and functional changes like this. It makes things harder to review and adds diff from upstream, which is a source of merge conflicts when synchronizing. Can you go through the commits in this PR and remove the intermixing of whitespace and functional changes within a single commit?

If you're going to fix whitespace nits, I suggest doing it upstream first, since they are not bugs, but style issues, and we want to keep it simple when it comes to avoiding merge conflicts when syncing with upstream later. That means minimizing diff from upstream.

Reply:
:+1:

Reply:
I see that I might need to look this one over again.

Reply:
Should be OK.